1.7/usage/tutorials/cassandra/index.md

```diff
@@ -146,13 +146,13 @@ core@ip-10-0-6-153 ~ $
 You are now inside your DC/OS cluster and can connect to the Cassandra cluster directly. Connect to the cluster using the cqlsh client:
 
 ```bash
-core@ip-10-0-6-153 ~ $ docker run cassandra:2.2.5 cqlsh <HOST>
+core@ip-10-0-6-153 ~ $ docker run -ti cassandra:latest cqlsh <HOST>
```

Comment:
ususally go with `-it`, as it sounds better when you read it out loud.  Not sure we maintain a `:latest` docker image...  Did you confirm what version that actually is?